model/src/main/java/com/basistech/rosette/dm/LayoutRegion.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2019 Basis Technology Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.basistech.rosette.dm;
+
+
+import com.google.common.base.MoreObjects;
+
+import java.io.Serializable;
+import java.util.Map;
+
+/**
+ * Layout defines text as spans defined by structured or unstructured regions.
+ */
+public class LayoutRegion extends Attribute implements Serializable {
+
+    public enum Layout {
+        UNSTRUCTURED,
+        STRUCTURED,
+    }
+
+    private static final long serialVersionUID = 250L;
```

Comment:
The package documentation says “Each class has a `serialVersionId`, [sic] the ID is derived from the version number of the library in which the a [sic] change was made.” Since the upcoming version is 2.7.0 this should be `270L`.